modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -139,6 +141,26 @@ public void deleteOrganizationAccountsByExternalReferenceCode(
 		}
 	}
 
+	@Override
+	public void
+			deleteOrganizationByExternalReferenceCodeAccountsByExternalReferenceCode(
+				String externalReferenceCode, String[] externalReferenceCodes)
+		throws Exception {
+
+		com.liferay.portal.kernel.model.Organization organization =
+			_organizationService.fetchOrganizationByExternalReferenceCode(
+				externalReferenceCode, contextCompany.getCompanyId());
+
+		if (organization == null) {
+			throw new NoSuchOrganizationException(
+				"Unable to find organization with external reference code " +
+					externalReferenceCode);
+		}
```

Comment:
can you check if for the organization we have the getOrganizationByExternalReferenceCode? if not let's evaluate if add it or go with this fetch + throw

Reply:
@smottal yep, we have the get version :)